streams-contrib/streams-provider-twitter/src/main/java/org/apache/streams/twitter/provider/TwitterFollowingProvider.java

```diff
@@ -129,7 +134,7 @@ public StreamsResultSet readCurrent() {
 
     @Override
     public void prepare(Object o) {
```

Comment:
It's delegating prepare() to it's parent class and opting to use the configuration provided by the constructor rather than by the streams runtime.  prepare(Object) is defined by the interface and thus has to be implemented, but exact implementation details vary.